core/templates/base-components/base-content.component.css

```diff
@@ -92,3 +92,9 @@
     margin: 0 auto;
   }
 }
+/* Fix #15530: Text in navbar overflows in smaller screen size; added media query to hide text in smaller screen size*/
+@media(min-width:1052px) {
+  .hide {
+    display: flex !important;
```

Comment:
Is this `!important` tag necessary? If so, please add a comment explaining it.

Reply:
Removed important as it was unnecessary and we can achieve the same by changing the class name because it was already used. @Lawful2002  PTAL